pages/common/views/base.jsx

```diff
@@ -3,6 +3,7 @@ import omit from 'lodash/omit';
 import { createStore } from 'redux';
 import { Provider } from 'react-redux';
 import HomeOffice from '@asl/service/ui/components/home-office';
+import rootReducer from '@asl/service/ui/reducers';
```

Comment:
Next question... if all of the things that this template needs to render exist in `@asl/service/ui`, why is this template not also there?

a.k.a. Make `@asl/pages` _only_ be pages.

Reply:
I was trying to stay in my lane

Reply:
you'd be a terrible driving instructor

Reply:
Or rather, by default `@asl/service` uses https://github.com/UKHomeOffice/asl-service/blob/master/ui/views/layout.jsx as the layout, which we override to use this template, so we can add all the custom stores and reducers.

Except now those aren't _custom_ stores and reducers, they're just stores and reducers, so why not make that file this file?

Reply:
Also, I'm happy to merge this, and then do what I describe later, but it seems a logical next step.

What _is_ your lane btw?

Reply:
my lane was the console errors and broken-ness on the dashboard

Reply:
and yeah, I'm totally for pulling everything that isn't a page out of pages, especially page.js

Reply:
Do you want me to merge this then, or will you crack on? Or both?

Reply:
I'd rather get this in, as it should fix the dashboard issue, then I will do the things

Reply:
as the things are likely to break the other things